decentraland/unity-renderer
# fix: Remove missing references on startup

## What does this PR change?

Cleanup PR. 
1) Empty folders .meta were still lurking in the project. When cloning , this would generate a warning on the console; and create as many empty folders as .meta were in the project. These orphans .meta were deleted
2) There was `missing references warnings` on initialization. This was caused by missing references files in the BuilderInWorld prefab steps tutorial. They were referenced by TutorialSettings, and when initialized, this warning was logged. Since this tutorial is not used anymore, missing references were taken away to achieve a cleaner startup.

## How to test the changes?

1. Run in Unity Editor. No warning message related to missing references should appear in the console.
2. Check that the .meta that were deleted are indeed deleted and no extra folder was created.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
